recipes/shovill/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
```

Comment:
You are not compiling here, but you use gcc as run time and build time dependency.
Do you really need to compile anything? If you just copy, you probably don't need anything as build time dependency.
Also zlib is probably not needed.